src/create-db/download-db-dependencies.ps1

```diff
@@ -0,0 +1,55 @@
+param(
+    [PSCustomObject[]]$dependencies
+)
+
+if ($null -eq $dependencies -or !$dependencies.PSobject.Properties.name.Contains("Count" ) -or $dependencies.Count -eq 0)
+{
+    return
+}
+
+Write-Host "Downloading database objects"
+
+$nugetFolder = "$PSScriptRoot\.nuget"
+$dependencyOutputFolder = "$PSScriptRoot\.dependencies"
+$targetNugetExe = "$nugetFolder\nuget.exe"
+$sourceNugetExe = "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe"
+
+if (![System.IO.File]::Exists($targetNugetExe))
+{
+    Write-Host "Downloading nuget.exe"
+    New-Item -ItemType Directory -Path $nugetFolder
+    Invoke-WebRequest $sourceNugetExe -OutFile $targetNugetExe
+}
```

Comment:
The self-hosted and github-hosted windows runners have nuget on them. so i don't think we need it for those machines, but I don't think this is going to work for linux machines.  Have you switched 1up over to linux to try this out?

Reply:
1Up doesn't have dependencies so this isn't run for it. I thought I had checked it out with a different repo, but now you've got me questioning myself. I'll go test that and report back.

Reply:
That sent me down quite the rabbit hole, but I'm back finally! So I think this nuget stuff was leftovers from some other work because it wasn't used. I removed it. I also tested on linux to make sure we could still download the nupkg files and unzip them. [It worked](https://github.com/bc-swat/database/runs/4553358974?check_suite_focus=true#step:6:103)!